.mci.yml

```diff
@@ -354,6 +354,10 @@ functions:
                   set -o errexit
                   set -o xtrace
 
+                  # TODO CXX-2227: Remove export after minimum libmongoc version is bumped.
+                  # export MONGOC_VERSION for `test_bson` test binary.
+                  export MONGOC_VERSION="${mongoc_version|master}"
```

Comment:
I added this env. variable to condition the two tests on the underlying libmongoc version. We can eventually remove this logic on any branch with libmongoc minimum version >= 1.18.0 (CXX-2227).